packages/salesforcedx-vscode-apex/snippets/apex.json

```diff
@@ -210,7 +210,7 @@
     "constructor": {
         "prefix": "constructor",
         "body": [
-            "${1:|private,public,global|} ${2:${TM_FILENAME_BASE}}($3) {",
+            "${1|private,public,global|} ${2:${TM_FILENAME_BASE}}($3) {",
```

Comment:
were these changes intentional?

Reply:
Oh i think this may have gotten picked up from develop, looks like it's a customer contribution